src/services/fileStorage/strategies/awsS3.js

```diff
@@ -97,6 +98,23 @@ if (Configuration.get('FEATURE_MULTIPLE_S3_PROVIDERS_ENABLED') === false) {
 }
 // end legacy
 
+const getS3 = (storageProvider) => {
+	const S3_KEY = Configuration.get('S3_KEY');
+	storageProvider.secretAccessKey = CryptoJS.AES.decrypt(storageProvider.secretAccessKey, S3_KEY).toString(
+		CryptoJS.enc.Utf8
+	);
+	return new aws.S3(getConfig(storageProvider));
+};
+
+const listBuckets = async (awsObject) => {
+	try {
+		const response = await awsObject.s3.listBuckets().promise();
+		return response.Buckets ? response.Buckets.map((b) => b.Name) : [];
+	} catch (e) {
+		throw new GeneralError(e);
```

Comment:
Please removethe try catch - it should be automatically handled by the express middleware (errorHandler)

Reply:
done